SimplenoteUITests/AllNotes.swift

```diff
@@ -5,29 +5,35 @@ class AllNotes {
     class func openNote(noteName: String) {
         app.tables.cells[noteName].tap()
     }
-    
+
     class func isOpen() -> Bool {
         return app.navigationBars[uidNavBar_AllNotes].exists
     }
 
-    class func open() {        
+    class func open() {
         guard !isOpen() else { return }
-        
+
         app.navigationBars.element.buttons[uidButton_Menu].tap()
         app.tables.staticTexts[uidButton_AllNotes].tap()
     }
 
     class func addNoteTap() {
-        print(">>> Adding new note...")
         app.navigationBars[uidNavBar_AllNotes].buttons[uidButton_NewNote].tap()
     }
 
-    class func createNoteAndLeaveEditor (noteName: String) {
+    class func createNoteAndLeaveEditor(noteName: String) {
+        print(">>> Creating note: " + noteName)
         AllNotes.addNoteTap()
         NoteEditor.clearAndEnterText(enteredValue: noteName)
         NoteEditor.leaveEditor()
     }
 
+    class func createNotes(noteNamesArray: Array<String>) {
```

Comment:
Another nitpick.

The [Swift API Guidelines](https://swift.org/documentation/api-design-guidelines/#weak-type-information) recommend to precede each weakly typed parameter, such as `Any`, `Int`, or `String, with a noun describing its role:

<img width="500" alt="screen shot 2019-02-06 at 1 43 58 pm" src="https://user-images.githubusercontent.com/1218433/52317755-78bf6600-2a15-11e9-93ba-ec4a22fb71cb.png">

In this case, `Array<String>` is clear enough that appending "array" to the name is note necessary.

What do you think of this name?

```suggestion
    class func createNotes(names: Array<String>) {
```

Reply:
Thanks! Applied